src/features/client/pages/RestaurantInspect/index.jsx

```diff
@@ -73,11 +76,29 @@ function RestaurantInspect() {
           </Button>
         </Flex>
       </Box>
-      <Box bg="white" paddingTop="20px">
-        {restaurantInfo.restaurantMenu.map((dealSection, i) => {
-          return <RestaurantInspectSection key={i} dealSection={dealSection} />;
-        })}
-      </Box>
+      <Flex padding={{ base: 0, sm: '0 10px' }} bg="white" gap="10px">
+        <Box bg="white" paddingTop="20px" width="100%" flex="6">
+          {restaurantInfo.restaurantMenu.map((dealSection, i) => {
+            return (
+              <RestaurantInspectSection
+                key={i}
+                dealSection={dealSection}
+                cartOpened={cartItems.length ? true : false}
```

Comment:
Move logic to component, this should only pass a value

Reply:
fixed